src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -246,7 +246,7 @@ private bool InitializeSearcherManager(IndexingConfiguration config, Directory d
                     shouldRetry: e =>
                     {
                         // Retry on any exception (but log it)
-                        _logger.LogError("Startup: An error occurred initializing searcher manager. Going to retry...",
+                        _logger.LogError("Startup: An error occurred initializing searcher manager. Going to retry... Exception: {Exception}",
                             e);
                         _searchTelemetryClient.TrackMetric(SearchTelemetryClient.MetricName.SearchIndexReopenFailed, 1);
```

Comment:
Does this write exceptions to the `exceptions` AI table? I thought you had to call `TrackException` or something like that.